From 9ea7db72cf70f11c5c627969daab86f69eea57b3 Mon Sep 17 00:00:00 2001
From: Matthew Waters <matthew@centricular.com>
Date: Sun, 27 Mar 2022 14:39:11 +1100
Subject: [PATCH 1/1] harfbuzz: disable werror from pragmas

Currently fails building on macOS 12.3 and should actually be controlled
by the build system.
---
 src/hb.hh | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/src/hb.hh b/src/hb.hh
index 558fea495..0b53a07aa 100644
--- a/src/hb.hh
+++ b/src/hb.hh
@@ -62,6 +62,8 @@
 
 /* Error.  Should never happen. */
 #ifndef HB_NO_PRAGMA_GCC_DIAGNOSTIC_ERROR
+#if 0
+#pragma GCC diagnostic error   "-Wc++11-narrowing"
 #pragma GCC diagnostic error   "-Wcast-align"
 #pragma GCC diagnostic error   "-Wcast-function-type"
 #pragma GCC diagnostic error   "-Wdelete-non-virtual-dtor"
@@ -93,6 +95,7 @@
 #pragma GCC diagnostic error   "-Wvla"
 #pragma GCC diagnostic error   "-Wwrite-strings"
 #endif
+#endif
 
 /* Warning.  To be investigated if happens. */
 #ifndef HB_NO_PRAGMA_GCC_DIAGNOSTIC_WARNING
-- 
2.38.1

